### PR TITLE
Provide go template to display in a concise way ClusterTestSuite

### PR DESCRIPTION
**Pull request by aszecowka**
- template provided
- Octopus logo!

### CODE REVIEW

File: docs/kubectl-integration.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Cluster Test Suite Template
+
+Status of a `ClusterTestSuite` contains many information.
+Output from following command:
+```
+kubectl get cts -oyaml {suite mame}
+```
+can be overwhelming. To get more concise but still informative output, you can use template stored in `kubectl/suite.template`.
+```
```
Comment (mszostok) [inline docs/kubectl-integration.md]: IMO we need to provide the command for downloading this template. 
```
curl -LO https://raw.githubusercontent.com/kyma-incubator/octopus/master/kubectl/suite.template
```

**Review by mszostok — APPROVED** [review]
LGTM

File: kubectl/suite.template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    {{- range .executions}}
+        {{- if eq .podPhase "Succeeded"}}+
+        {{- else if eq .podPhase "Failed"}}-
+        {{- else }}?
+        {{- end}}
+    {{- end}}
+    {{- $ex:=len .executions}}
+    {{- if eq $ex 0}}?
+    {{- end}}
+{{- end}}
```
Comment (mszostok) [inline kubectl/suite.template]: please add new line

File: docs/kubectl-integration.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Name:           {suite name}
+Concurrency:    1.0
+MaxRetries:     1.0
+Count:          1.0
+StartTime:      2019-04-06T12:56:11Z
+CompletionTime: 2019-04-06T13:22:38Z
+Condition:      Failed
+Tests:
+    test-monitoring - +
```
Comment (mszostok) [inline docs/kubectl-integration.md]: please describe what `- +` means

File: README.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # Octopus
 
+![](docs/assets/octopus.JPG)
```
Comment (klaudiagrz) [inline README.md:3]: I love it! 😍 🐙

File: README.md [hunk 5]
```diff
@@ -9,6 +11,8 @@ It was created to replace the `helm test` by providing these additional features
 - Running tests multiple times, which is helpful when you add a new test and you want to validate if it is stable, or when you want to reproduce a problem that occurs on CI.
 - Full support for concurrent testing, which allows you to specify how many tests should be executed at the same time.
 
+Octopus provides integration with `kubectl`. For more info, read[kubectl integration](./docs/kubectl-integration.md)
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
Octopus provides integration with `kubectl`. For more information, read [this](./docs/kubectl-integration.md) document.
```

File: docs/README.md [hunk 6]
```diff
@@ -7,6 +7,7 @@ The folder contains documents that provide insight into Octopus functionality:
 <!-- Update the list each time you modify the document structure in this folder. -->
 
 - [Define test and run test suite](tutorial.md)
```
Comment (klaudiagrz) [inline docs/README.md]:
```suggestion
- [Tutorial on how to define test and run test suite](tutorial.md) 
```

File: docs/kubectl-integration.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+# Integration with Kubectl
+
+Octopus provides integration with `kubectl` to simplify working with it's resources.
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
Octopus provides integration with `kubectl` to simplify the work on its resources.
```

File: docs/kubectl-integration.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+## Cluster Test Suite Template
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
## ClusterTestSuite template
```

File: docs/README.md [hunk 9]
```diff
@@ -7,6 +7,7 @@ The folder contains documents that provide insight into Octopus functionality:
 <!-- Update the list each time you modify the document structure in this folder. -->
 
 - [Define test and run test suite](tutorial.md) 
+- [Kubectl integration](kubectl-integration.md)
```
Comment (klaudiagrz) [inline docs/README.md]:
```suggestion
- [Kubectl integration guide](kubectl-integration.md)
```

File: docs/kubectl-integration.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+Status of a `ClusterTestSuite` contains many information.
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]: To get the full status of ClusterTestSuite, run: 
```
kubectl get cts -oyaml {suite mame}
```

However, the output of this status contains a lot of information, which can be overwhelming. To get more concise but still informative output, use the template stored in `kubectl/suite.template`. Run:

File: docs/kubectl-integration.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+Output from following command:
+```
+kubectl get cts -oyaml {suite mame}
+```
+can be overwhelming. To get more concise but still informative output, you can use template stored in `kubectl/suite.template`.
+```
+ kubectl get cts {suite name} -ogo-template-file --template={path to template file}
+```
+
+Output:
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
The simplified output looks as follows:
```

File: docs/kubectl-integration.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+StartTime:      2019-04-06T12:56:11Z
+CompletionTime: 2019-04-06T13:22:38Z
+Condition:      Failed
+Tests:
+    test-monitoring - + 
+    test-events +
+    test-ui ?
+```
+
+Next to the test names, status of the test executions is displayed:
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
Next to the test name, there is a symbol which specifies the status of a given test execution:
```

**Review by klaudiagrz — COMMENTED** [review]
General remark: shouldn't we change the name of the file, for example into `ClusterTestSuite status` ? I might be wrong but imo it would be more precise.

File: README.md (hunk 4 above)
Comment (aszecowka) [reply README.md:3]: 🐙

File: docs/kubectl-integration.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
+# Integration with Kubectl
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
# Integration with kubectl
```

File: docs/kubectl-integration.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+## ClusterTestSuite template
+
+To get the full status of ClusterTestSuite, run:
+```
+kubectl get cts -oyaml {suite mame}
+```
+However, the output of this status contains a lot of information, which can be overwhelming. 
+To get more concise but still informative output, use the template stored in `kubectl/suite.template`. 
+To download recent version of the template, execute the following command:
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
To download the recent version of the template, run:
```

File: docs/kubectl-integration.md [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+```
+curl -LO https://raw.githubusercontent.com/kyma-incubator/octopus/master/kubectl/suite.template
+```
+Then, set the `template` argument to point to the downloaded file:
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
Then, set the `--template` argument to point to the downloaded file:
```

File: README.md [hunk 16]
```diff
@@ -9,6 +11,8 @@ It was created to replace the `helm test` by providing these additional features
 - Running tests multiple times, which is helpful when you add a new test and you want to validate if it is stable, or when you want to reproduce a problem that occurs on CI.
 - Full support for concurrent testing, which allows you to specify how many tests should be executed at the same time.
 
+Octopus provides integration with `kubectl`. For more information, read [this](./docs/kubectl-integration.md) document.
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
Octopus provides integration with kubectl. For more information, read [this](./docs/kubectl-integration.md) document.
```

File: docs/kubectl-integration.md [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+    test-events +
+    test-ui ?
+```
+
+Next to the test name, there is a symbol which specifies the status of a given test execution:
+- `+` - test execution passed
+- `-` - test execution failed
+- `?` - test execution is in progress or not yet scheduled
+
+In the above example, `test-monitoring` failed at the first time, then it was retried and finally it succeeded.
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
>**NOTE:** In the given example, `test-monitoring` failed at first, then it was retired, and finally it succeeded.  
```

File: docs/kubectl-integration.md [hunk 18]
```diff
@@ -0,0 +1,41 @@
+# Integration with Kubectl
+
+Octopus provides integration with `kubectl` to simplify the work on its resources.
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
Octopus provides integration with kubectl to simplify work on its resources.
```

File: docs/kubectl-integration.md [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+
+## ClusterTestSuite template
+
+To get the full status of ClusterTestSuite, run:
```
Comment (klaudiagrz) [inline docs/kubectl-integration.md]:
```suggestion
You can get the full status of ClusterTestSuite by running:
```

File: docs/README.md [hunk 20]
```diff
@@ -6,9 +6,10 @@ The folder contains documents that provide insight into Octopus functionality:
 
 <!-- Update the list each time you modify the document structure in this folder. -->
 
-- [Define test and run test suite](tutorial.md) 
+- [Tutorial on how to define test and run test suite](tutorial.md)
```
Comment (klaudiagrz) [inline docs/README.md]:
```suggestion
- [Tutorial on how to define a test and run a test suite](tutorial.md) 
```

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**